frontend/src/app/libs/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { IUser } from '../container/models/user';
+import { SessionFacadeService } from '../container/store/session/session.facade';
+
+@Component({
+	selector: 'app-sign-in',
+	templateUrl: './sign-in.component.html',
+	styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent {
+	public user: IUser;
+
+	public signIn: FormGroup = new FormGroup({
+		email: new FormControl('', [Validators.required, Validators.email]),
+		password: new FormControl('', [Validators.required]),
+	});
+
+	constructor(private sessionFacadeService: SessionFacadeService) {}
+
+	public submit(): void {
+		this.user = this.signIn.value;
```

Comment:
Is `signIn` value is really user? Seems like you just store username and password here, it;s not user, so create more meaningfull name like something with `interface UserCredentials { login: string, password: string }` and current this.user will become `this.credentials` or `this.userCredentials`